<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="7" passed="4" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-06-25T23:42:48 CEST" name="e-Bankingv1" finished-at="2021-06-25T23:44:05 CEST" duration-ms="76848">
    <groups>
    </groups>
    <test started-at="2021-06-25T23:42:48 CEST" name="BankingApp Test" finished-at="2021-06-25T23:44:05 CEST" duration-ms="76848">
      <class name="com.banking.testCases.TC_AddCustomerTest_003">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.banking.testCases.TC_AddCustomerTest_003@7193666c]" started-at="2021-06-25T23:43:44 CEST" name="setup" finished-at="2021-06-25T23:43:49 CEST" duration-ms="4469" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="addNewCustomer()[pri:0, instance:com.banking.testCases.TC_AddCustomerTest_003@7193666c]" started-at="2021-06-25T23:43:49 CEST" name="addNewCustomer" finished-at="2021-06-25T23:44:05 CEST" duration-ms="15889" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewCustomer -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.banking.testCases.TC_AddCustomerTest_003@7193666c]" started-at="2021-06-25T23:44:05 CEST" name="tearDown" finished-at="2021-06-25T23:44:05 CEST" duration-ms="128" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.banking.testCases.TC_AddCustomerTest_003 -->
      <class name="com.banking.testCases.TC_LoginTest_001">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.banking.testCases.TC_LoginTest_001@243c4f91]" started-at="2021-06-25T23:42:48 CEST" name="setup" finished-at="2021-06-25T23:42:59 CEST" duration-ms="11182" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginTest()[pri:0, instance:com.banking.testCases.TC_LoginTest_001@243c4f91]" started-at="2021-06-25T23:42:59 CEST" name="loginTest" finished-at="2021-06-25T23:43:02 CEST" duration-ms="2325" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.banking.testCases.TC_LoginTest_001@243c4f91]" started-at="2021-06-25T23:43:02 CEST" name="tearDown" finished-at="2021-06-25T23:43:02 CEST" duration-ms="176" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.banking.testCases.TC_LoginTest_001 -->
      <class name="com.banking.testCases.TC_LoginDDT_002">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.banking.testCases.TC_LoginDDT_002@5119fb47]" started-at="2021-06-25T23:43:02 CEST" name="setup" finished-at="2021-06-25T23:43:07 CEST" duration-ms="5244" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginDTT(java.lang.String,java.lang.String)[pri:0, instance:com.banking.testCases.TC_LoginDDT_002@5119fb47]" started-at="2021-06-25T23:43:13 CEST" name="loginDTT" data-provider="LoginData" finished-at="2021-06-25T23:43:22 CEST" duration-ms="9009" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr336603]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[aqetYje]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDTT -->
        <test-method signature="loginDTT(java.lang.String,java.lang.String)[pri:0, instance:com.banking.testCases.TC_LoginDDT_002@5119fb47]" started-at="2021-06-25T23:43:22 CEST" name="loginDTT" data-provider="LoginData" finished-at="2021-06-25T23:43:27 CEST" duration-ms="4407" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr83460]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qAbUzyj]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at com.banking.testCases.TC_LoginDDT_002.loginDTT(TC_LoginDDT_002.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDTT -->
        <test-method signature="loginDTT(java.lang.String,java.lang.String)[pri:0, instance:com.banking.testCases.TC_LoginDDT_002@5119fb47]" started-at="2021-06-25T23:43:27 CEST" name="loginDTT" data-provider="LoginData" finished-at="2021-06-25T23:43:31 CEST" duration-ms="4425" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr137319]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[bAdYvyb]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at com.banking.testCases.TC_LoginDDT_002.loginDTT(TC_LoginDDT_002.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDTT -->
        <test-method signature="loginDTT(java.lang.String,java.lang.String)[pri:0, instance:com.banking.testCases.TC_LoginDDT_002@5119fb47]" started-at="2021-06-25T23:43:31 CEST" name="loginDTT" data-provider="LoginData" finished-at="2021-06-25T23:43:40 CEST" duration-ms="8651" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr336603]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[aqetYje]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDTT -->
        <test-method signature="loginDTT(java.lang.String,java.lang.String)[pri:0, instance:com.banking.testCases.TC_LoginDDT_002@5119fb47]" started-at="2021-06-25T23:43:40 CEST" name="loginDTT" data-provider="LoginData" finished-at="2021-06-25T23:43:44 CEST" duration-ms="4428" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr112909]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[tytUreg]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at com.banking.testCases.TC_LoginDDT_002.loginDTT(TC_LoginDDT_002.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDTT -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.banking.testCases.TC_LoginDDT_002@5119fb47]" started-at="2021-06-25T23:43:44 CEST" name="tearDown" finished-at="2021-06-25T23:43:44 CEST" duration-ms="175" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.banking.testCases.TC_LoginDDT_002 -->
    </test> <!-- BankingApp Test -->
  </suite> <!-- e-Bankingv1 -->
</testng-results>
